rotkehlchen/exchanges/manager.py

```diff
@@ -299,7 +303,10 @@ def get_all_binance_pairs(self) -> List[str]:
         if Location.BINANCEUS in self.connected_exchanges:
             binance = cast('Binance', self.connected_exchanges[Location.BINANCEUS][0])
             return list(binance.symbols_to_pair.keys())
-        return []
+        pairs = list(query_binance_exchange_pairs().keys())
```

Comment:
There is no need for `get_all_binance_pairs()` to exist here anymore since it does not need the binance object. It should just go away.